src/app/api/tickets/route.ts

```diff
@@ -0,0 +1,37 @@
+import { NextRequest, NextResponse } from "next/server";
+import { PrismaClient, RequestStatus } from "@prisma/client"; // Import RequestStatus enum
+
+const prisma = new PrismaClient(); // Initialize PrismaClient
```

Comment:
We can import the instance of the prisma client from here:
`src/lib/prisma.ts`

instead of creating a new one :)